docs/agents/sematext-agent/windows-installation.md

```diff
@@ -0,0 +1,142 @@
+
+To see your Windows host metrics in Sematext Cloud, you need to [create an Infra App](https://sematext.com/docs/monitoring/infrastructure/#create-an-infra-app) and select the Windows environment. This will provide you with an Infra App token, which is essential for establishing a secure link between Sematext Cloud and Sematext Agent.
+
+## Install from GUI
+
+1. Download the [Sematext Agent installer](https://todo-add-public-installer-repo)
+2. Run the installer as an Administrator
+3. Follow the installation steps and accept the license agreement 
+4. Set the Infra App token
+5. Select a region (EU or US) based on where you created your account in Sematext Cloud
+
+After completing these steps, Sematext Agent will be automatically started as a Windows Service.
+
+
+## Install from command line
+
+Download the [Sematext Agent installer](https://todo-add-public-installer-repo).
+
+### Using PowerShell
+
+Run the following command as Administrator from the folder that you downloaded the installer:
+
+```
+    Start-Process -Wait msiexec -ArgumentList "/qn /i $($msiFileName = 'sematext-agent-VERSION.msi'; $msiFileName) /L*V `"$msiFileName.log`" INFRA_APP_TOKEN=YOUR_INFRA_TOKEN REGION=YOUR_REGION_EU_or_US"
+```
+
+Where:
+
+- `YOUR_INFRA_APP_TOKEN`: the Infra App token
+- `YOUR_REGION_EU_or_US`: your Sematext Cloud region
+- `VERSION`: the agent version
+
+For example:
+
+```
+    Start-Process -Wait msiexec -ArgumentList "/qn /i $($msiFileName = 'sematext-agent-3.3.0.2.msi'; $msiFileName) /L*V `"$msiFileName.log`" INFRA_APP_TOKEN=7511db7f-c060-4e10-b667-5f2653d4933e REGION=EU"
+```
+
+### Using Windows CMD:
+
+Run the following command as Administrator from the folder that you downloaded the installer:
+
+```
+    start /wait msiexec /qn /i "sematext-agent-VERSION.msi" /L*V "sematext-agent-VERSION.msi.log" INFRA_TOKEN=YOUR_INFRA_APP_TOKEN REGION=YOUR_REGION_EU_or_US
+```
+
+Where:
+
+- `YOUR_INFRA_APP_TOKEN`: the Infra App token
+- `YOUR_REGION_EU_or_US`: your Sematext Cloud region
+- `VERSION`: the agent version
+
+For example:
+
+```
+    start /wait msiexec /qn /i "sematext-agent-3.3.0.2.msi" /L*V "sematext-agent-3.3.0.2.msi.log" INFRA_APP_TOKEN=7511db7f-c060-4e10-b667-5f2653d4933e REGION=EU
+```
+
+After completing these steps, Sematext Agent will be automatically started as a Windows Service.
+
+## How to check the Sematext Agent service status
+
+### Windows Services
+
+Open Windows Services and find the Sematext Agent service:
+
+![Sematext Agent service](images/sematext-agent-service.png)
+
+### Command line
+
+Run the following command:
+
+```
+cd  'C:\Program Files\Sematext Agent\' .\st-agent-amd64.exe windows-service status
+```
+
+## How to start / stop / restart Sematext Agent
+
+Run one of the following commands to start, stop or restart Sematext Agent
```

Comment:
You may want to mention that it's possible to do that via GUI (Windows Services application)